src/accelerate/components/List.tsx

```diff
@@ -94,6 +111,15 @@ export default function List( props: Props ) {
 								timer = setTimeout( value => {
 									setSearch( value );
 								}, 500, e.target.value );
+								analytics.track(
+									'filter',
+									{
+										location: 'dashboard',
+										filter_type: 'search',
+										filter_value: e.target.value,
+									}
+								);
```

Comment:
Lets put this inside the `setTimeout` callback or we're going to be tracking events on every keystroke